ts_src/script.ts

```diff
@@ -200,14 +200,19 @@ export function toASM(chunks: Uint8Array | Array<number | Uint8Array>): string {
 export function fromASM(asm: string): Uint8Array {
   v.parse(v.string(), asm);
 
+  // Compile the ASM string into a Uint8Array
   return compile(
-    asm.split(' ').map(chunkStr => {
-      // opcode?
-      if (OPS[chunkStr] !== undefined) return OPS[chunkStr];
-      v.parse(types.HexSchema, chunkStr);
+    asm.split(' ').map((chunk: string): number | Uint8Array => {
+      // Check if the chunk is an opcode
+      if (chunk in OPS) {
```

Comment:
This is subtly different.

"81 82 83 84 85" should output 5 PUSHDATAs but because of the embedded REVERSE_OPS it will come out as the same as "OP_1 OP_2 OP_3 OP_4 OP_5"